src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -216,6 +216,9 @@
   // validate the default parameters by GS?
   private boolean validateDefaultParameters;
 
+  // threshold for uploading file in multiple chunks for put commands. Default 200MB
+  private int multipartUploadThreshold = 200;
```

Comment:
This should be given from the GS at all times, right?

Reply:
yes, that's true. I could take it out here